content/master/getting-started/provider-aws-part-2.md

```diff
@@ -0,0 +1,26 @@
+---
+title: AWS Quickstart Part 2
+weight: 120
+tocHidden: true
+---
+
+{{< hint "important" >}}
+This guide is part 2 of a series. Follow **[part 1]({{<ref "provider-aws" >}})** 
+to install Crossplane and connect your Kubernetes cluster to AWS.
+{{< /hint >}}
+
+This section creates a _Composition_, _Custom Resource Definition_ and _Claim_
+to create a custom Kubernetes API to create AWS resources. 
+
+## Prerequisites
+* Complete [quickstart part 1]({{<ref "provider-aws" >}}) connecting Kubernetes
+  to AWS.
+* an AWS account with permissions to create EC2 compute instances
+
+
+## Create a Composition
+[Part 1]({{<ref "provider-aws" >}}) created a single _managed resource_.
+A _Composition_ is a template to create multiple _managed resource_ at the same
+time.
+
+In this example we will create 
```

Comment:
more to this AWS part 2 page is coming? 

I came here because i was curious what the experience would be now without the content under snippets.  in the current docs, we show the content that people can copy/paste, but also give them a quick command like:

```console
kubectl apply -f https://raw.githubusercontent.com/crossplane/crossplane/master/docs/snippets/compose/claim-aws.yaml
```

Everything will be inline copy/pasteable with this new docs reorg flow, do I have that right?

Reply:
the rest of part 2 is being written right now.

I'm not opposed to a single package or feature pull like that but part of the goal is to walk through what is being configured and what it does. I'd prefer to show the yaml and give them an easy copy/paste experience.

Reply:
Follow up: [Part 2](https://deploy-preview-271--crossplane.netlify.app/v1.11/getting-started/provider-aws-part-2/) and [Part 3](https://deploy-preview-271--crossplane.netlify.app/v1.11/getting-started/provider-aws-part-3/) are completed.